apollo-client/src/main/java/com/ctrip/framework/apollo/monitor/internal/listener/impl/DefaultApolloClientNamespaceApi.java

```diff
@@ -0,0 +1,241 @@
+    createOrUpdateCounterSample(mapKey, ApolloClientMonitorConstant.METRICS_NAMESPACE_USAGE,
+        Collections.singletonMap(NAMESPACE, namespace), 1);
+  }
+
+  private void handleUpdateTimeEvent(ApolloClientMonitorEvent event,
+      NamespaceMetrics namespaceMetrics) {
+    long updateTime = event.getAttachmentValue(ApolloClientMonitorConstant.TIMESTAMP);
+    namespaceMetrics.setLatestUpdateTime(updateTime);
+  }
+
+  private void handleFirstLoadSpendEvent(ApolloClientMonitorEvent event,
+      NamespaceMetrics namespaceMetrics) {
+    long firstLoadSpendTime = event.getAttachmentValue(ApolloClientMonitorConstant.TIMESTAMP);
+    namespaceMetrics.setFirstLoadSpend(firstLoadSpendTime);
+  }
+
+  private void handleReleaseKeyEvent(ApolloClientMonitorEvent event,
+      NamespaceMetrics namespaceMetrics) {
+    String releaseKey = event.getAttachmentValue(NAMESPACE_RELEASE_KEY);
+    namespaceMetrics.setReleaseKey(releaseKey);
+  }
+
+  @Override
+  public void export0() {
+    namespaces.forEach((namespace, metrics) -> {
+      // update NamespaceMetrics
+      createOrUpdateGaugeSample(namespace + METRICS_NAMESPACE_FIRST_LOAD_SPEND,
+          METRICS_NAMESPACE_FIRST_LOAD_SPEND,
+          Collections.singletonMap(NAMESPACE, namespace),
+          metrics.getFirstLoadSpend());
+
+      createOrUpdateGaugeSample(namespace + METRICS_NAMESPACE_ITEM_NUM,
+          METRICS_NAMESPACE_ITEM_NUM,
+          Collections.singletonMap(NAMESPACE, namespace),
+          m_configs.get(namespace).getPropertyNames().size());
+    });
+
+    //  update ConfigFile num
+    createOrUpdateGaugeSample(METRICS_CONFIG_FILE_NUM,
+        METRICS_CONFIG_FILE_NUM,
+        Collections.emptyMap(),
+        m_configFiles.size());
+
+    //  update NamespaceStatus metrics
+    createOrUpdateGaugeSample(METRICS_NAMESPACE_NOT_FOUND,
+        METRICS_NAMESPACE_NOT_FOUND,
+        Collections.emptyMap(),
+        namespace404.size());
+
+    createOrUpdateGaugeSample(METRICS_NAMESPACE_TIMEOUT,
+        METRICS_NAMESPACE_TIMEOUT,
+        Collections.emptyMap(),
+        namespaceTimeout.size());
+  }
+
+  @Override
+  public Map<String, NamespaceMetrics> getNamespaceMetrics() {
+    return Collections.unmodifiableMap(namespaces);
+  }
+
+  @Override
+  public List<String> getNotFoundNamespaces() {
+    return new ArrayList<>(namespace404);
+  }
+
+  @Override
+  public List<String> getTimeoutNamespaces() {
+    return new ArrayList<>(namespaceTimeout);
+  }
+
+  @Override
+  public Integer getNamespaceItemsNum(String namespace) {
+    Config config = m_configs.get(namespace);
+    return (config != null) ? config.getPropertyNames().size() : 0;
+  }
+
+  @Override
+  public Integer getConfigFileNum() {
+    return m_configFiles.size();
+  }
+
+  public static class NamespaceMetrics {
+
+    private int usageCount;
+    private long firstLoadSpend;
+    private long latestUpdateTime = System.currentTimeMillis();
```

Comment:
use LocalDateTime instead of long?

```suggestion
    private LocalDateTime latestUpdateTime = LocalDateTime.now();
```